docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,474 @@
+dapr-operator          dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-sidecar-injector  dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21  
+dapr-placement         dapr-system  True     Running  0.10.0    1d   2020-08-18 10:15.21 
+```
+
+After Dapr 0.10.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Multi Pub/Sub
+
+With the v0.10.0 release, we have added the support for multiple pub/sub components. That means that you can now publish and subscribe to topics arriving from different components.
+
+To this end, several changes have been made to both the publisher and consumer sides.
+
+#### Publisher
+
+An additional parameter has been added to tell Dapr which pub/sub component you're targeting.
+
+Old API:
+
+```
+POST http://localhost:<daprPort>/v1.0/publish/<topic>
+```
+
+New API:
+
+```
+POST http://localhost:<daprPort>/v1.0/publish/<pubsubname>/<topic>
+```
+
+#### Subscriber
+
+*Note: The following example in in Javascript but applicable to all languages*
+
+An additional `pubsubName` field has been added to the subscription payload to tell Dapr which pub/sub component this subscription is for.
+
+Old subscription:
+
+```javascript
+app.get('/dapr/subscribe', (req, res) => {
+    res.json([
+        {
+            topic: "newOrder",
+            route: "orders",
+        }
+    ]);
+})
+```
+
+New subscription:
+
+```javascript
+app.get('/dapr/subscribe', (req, res) => {
+    res.json([
+        {
+            topic: "newOrder",
+            route: "orders",
+            pubsubName: "messagebus"
+        }
+    ]);
+})
+```
+
+Changes have been made to the different SDKs, so make sure to get the latest version of the SDK you're using.
+Specifically, the subscription structures have added a `pubsubName` field and the publish methods take the `pubsubName` argument.
+
+#### CLI
+
+The `publish` command on the Dapr CLI now takes a required param `--pubsub` to specifiy the pubsub component:
+
+```
+dapr publish --pubsub pubsub --topic myevent --data '{ "name": "yoda" }'
+```
+
+### State retries
+
+The retry options for state operations have been removed. If you've never used retries, this is not a breaking change.
+The updated API references can be found [here](https://github.com/dapr/docs/blob/master/reference/api/state_api.md).
+
+### Java SDK
+
+The Service Invocation method now accepts an HTTPExtension argument.
+
+Old method call:
+
+```java
+client.invokeService(serviceAppId, method, message);
+```
+
+New method call:
+
+```java
+client.invokeService(serviceAppId, method, message, HttpExtension.NONE);
+```
+
+### CLI install path
+
+The command `dapr init` will now install the runtime binary `daprd` into `$HOME/.dapr/bin` for Mac/Linux and `%USERPROFILE%\.dapr\` for Windows. 
```

Comment:
Can we also mention that we removed the `--install-path` option: https://github.com/dapr/dapr/pull/1821/files#diff-de7b0f742b687d0de5db1bba55e17a66R28

Reply:
Added.